### PR TITLE
add test for contributor - for /info endpoint

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

adding test for /info endpoint to fix master which starts failing after merged with #1443 
upgrade-blob-lib-part-2-add-test-for-contributor

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                new BlobContainerItem().setName("sscs"),
+                new BlobContainerItem().setName("bulkscan"),
+                new BlobContainerItem().setName("sscs-rejected")
+                )
+            );
+
+        this.mockMvc.perform(get("/info"))
+            .andDo(print())
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.app.name").isNotEmpty())
```
Comment (doncem) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java:50]: we want to be sure of the app name. it plays significant role in app insights cloud role

Comment (doncem) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java:50]: although it's not part of class name test but it is part of application test

Comment (mdayican) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java:50]: these did not make sense for me 
`        "name": "Bulk Scanning",
        "description": "Ingest scanned documents."
`

Comment (doncem) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java:50]: oh dear. this needs fixing:

```yaml
info:
  app:
    name: Bulk Scanning
```

should be `Bulk Scan Processor`

that's what i mean - name should be checked. description i'm not so worry about right now

Comment (mdayican) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java]: changed the name

**Pull request merged**